### ISSUE TITLE
please open downloads for deb file

### ISSUE THREAD

**Issue by MontrealSergiy** [Comp-Labs/Google-Docs#18]
deb build is not available . Debian google doc app download link from https://complabs.in is dead for average user, perhaps all fine for github google doc app members. Try to open  it in anonymous browser mode

https://github.com/Comp-Labs/Google-Docs/releases/download/v2022.12.1/Google-Docs-amd64.deb

Sorry if you policy to ignore all not-codebase related issues, but your feedback form at https://complabs.in/contact is dead

**Comment by RudraSen2** [comment]
Yeah... I'm sorry if you're having problems with the website because I haven't updated it yet... This issue is caused because I've changed the file name of all the packages so I forgot to update the new links. Thanks! I'll update it. I was currently using Postform as the backend for the contact form but Postform is discontinued like a ghost... So I changed the Code in Local but have not published it yet... Thanks anyway!
